docs/task.md

```diff
@@ -0,0 +1,107 @@
+Tasks
+=====
+
+Task objects represent a user-created task on a file.
+
+
+Get a Task's Information
+------------------------
+
+Calling `task.get(fields=None, headers=None)` will return information about the specified task.
+
+```python
+task_info = client.task('1234').get()
+```
+
+List Tasks on File
+------------------
+
+To retrieve all tasks for a given file use, `file.get_tasks(limit=None, marker=None, fields=None)`.
+
+```python
+tasks = client.file('1111').get_tasks()
+for task in tasks:
+    # Do something
+```
+
+Add Task to File
+----------------
+
+To create a single task for a single user on a single file use, `file.create_task(message=None, due_at=None)`
+
+```python
+message = 'Please review this'
+due_at = "2014-04-03T11:09:43-07:00"
+task = client.file('11111').create_task(message, due_at)
+```
+
+Update Task Info
+----------------
+
+To update the task information use, `task.update_info(data, params=None, headers=None, **kwargs)`
```

Comment:
What might the user want to pass in for `params` or `headers`, and what are the `kwargs` used for?  These base class properties should either be better-documented or removed.